### PR TITLE
fix: set different WP domains for different netlify contexts - DO NOT MERGE

### PR DESCRIPTION
**Pull request by cindyli**
* [X] This pull request has been linted by running `npm run lint` without errors
* [X] This pull request has been tested by running `npm run dev` and reviewing affected routes
* [X] This pull request has been built by running `npm run generate` without errors
* [X] This isn't a duplicate of an existing pull request

## Description

To experiment how to set different WP domains for different netlify contexts.

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[context.deploy-preview.environment]
+	WP_DOMAIN = "https://wecount-dev.inclusivedesign.ca"
+
+[context.branch-deploy.environment]
+	WP_DOMAIN = "https://wecount-cms.inclusivedesign.ca"
```
Comment (greatislander) [inline netlify.toml:5]: Brilliant, @cindyli!

Comment (cindyli) [reply netlify.toml:5]: Unfortunately, it only works at the build time running `npm run generate` but not at the runtime when accessing webpages. I found `process.env.CONTEXT` is not picked up at the runtime too. It always returns `undefined` at browsing webpages.

Still in the process of figuring out how to pass environment variable values from the build time to runtime.

Comment (greatislander) [reply netlify.toml]: I think that is what this is supposed to do: https://nuxtjs.org/api/configuration-env/

Comment (cindyli) [reply netlify.toml]: Works perfectly. Thanks, @greatislander!

File: .eslintrc.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ module.exports = {
 		"quotes": [2, "double", { "avoidEscape": true }],
 		"import/no-named-as-default": 0,
 		"semi": [2, "always"],
-		"vue/html-indent": [1, "tab"]
+		"vue/html-indent": [1, "tab"],
+		"no-console": "off"
```
Comment (greatislander) [inline .eslintrc.js:23]: I'd rather we just add an ignore to the relevant lines in the source file than ignoring this altogether. See: https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments

Comment (cindyli) [reply .eslintrc.js:23]: "no-console" is turned off so I can debug the deploy preview. It won't be included in the pull request that will be merged into the dev and master.

Will close this pull request soon once the proper pull request is ready.

**Comment by cindyli** [comment]
Closed in favor of https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/205.

**Pull request closed without merging**